### PR TITLE
add ngprogress v1.1.1 with npm autoupdate config

### LINKED ISSUE #7427: [Request] Add ngProgress by VictorBjelkholm
Library name: ngProgress
Git repository url: https://github.com/victorbjelkholm/ngprogress
npm package url(optional):** 
License(s): The MIT License (MIT)
Official homepage: https://github.com/victorbjelkholm/ngprogress
Wanna say something? AngularJS module based on nprogress by rstacruz

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7427 
@PeterDaveHello Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/victorbjelkholm/ngprogress
 * Official website (optional, not the repository): https://github.com/victorbjelkholm/ngprogress
 * NPM package url (optional): https://www.npmjs.com/package/ngprogress
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1298
   - Count of watchers: 38
   - Count of forks: 189
 * NPM download stats (optional):
   - Downloads in the last day: 12
   - Downloads in the last week: 43
   - Downloads in the last month: 228

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7427, cc @VictorBjelkholm

### CODE REVIEW

File: ajax/libs/ngprogress/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "author": {
+    "name": "Victor Bjelkholm"
+  },
```
Comment (PeterDaveHello) [inline ajax/libs/ngprogress/package.json]: Let's use `"author": "Victor Bjelkholm"` here. You can take a look at its package.json in npm package.

Comment (pvnr0082t) [inline ajax/libs/ngprogress/package.json]: @PeterDaveHello Oh! sorry, I forgot to modify again.

File: ajax/libs/ngprogress/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "author": "Victor Bjelkholm",
+  "description": "slim, site-wide progressbar for AngularJS",
+  "homepage": "https://github.com/VictorBjelkholm/ngProgress#readme",
+  "keywords": [
+    "angular",
+    "ng-progress",
+    "ngProgress",
+    "angular",
```
Comment (PeterDaveHello) [inline ajax/libs/ngprogress/package.json]: Duplicated with line 6

Comment (pvnr0082t) [inline ajax/libs/ngprogress/package.json]: oh! I didn't notice this, thank you.

**Comment by ethan-f** [comment]
It looks like the most recent version 1.1.2 was not posted, but an older version of 1.1.1 which is broken.

**Comment by PeterDaveHello** [comment]
@pvnr0082t didn't you grab the files from its npm pacakge v1.1.1?

I'm not sure what's the reason why there is a difference about the newline char at EOF, maybe you touched it by your editor, there is a newline in your commit but didn't exist in its npm package, should be care about this next time:

```diff
< angular.module('ngProgress', ['ngProgress.directive', 'ngProgress.provider']);
---
> angular.module('ngProgress', ['ngProgress.directive', 'ngProgress.provider']);
\ No newline at end of file
```

@ethan-f though there is a newline char difference, but all other parts are the same as upstream, so you should ask for help from upstream, we have nothing to do with its content, sorry.

**Pull request merged**